src/algebra/order/sub.lean

```diff
@@ -670,6 +670,21 @@ lemma tsub_right_inj (hba : b ≤ a) (hca : c ≤ a) : a - b = a - c ↔ b = c :
 contravariant.add_le_cancellable.tsub_right_inj contravariant.add_le_cancellable
   contravariant.add_le_cancellable hba hca
 
+variables (α)
+
+/-- A `canonically_ordered_add_monoid` with ordered subtraction and order-reflecting addition is
+cancellative. This is not an instance at it would form a typeclass loop.
+
+See note [reducible non-instances]. -/
+@[reducible]
```

Comment:
I know this is not agreed upon, but I would expect the note to be very quite irrelevant to the documentation.
```suggestion
cancellative. This is not an instance at it would form a typeclass loop. -/
@[reducible] -- See note [reducible non-instances]
```

Reply:
Two reasons for keeping the note in the docstring:

* `@[reducible]` is (or should be) visible in doc-gen
* Having it in the docstring means the library note gets a backreference in the library note page.

Reply:
Regarding the last point, looking for `@[reducible] -- See note [reducible non-instances]` brings 45 results in 13 files.

Reply:
That may be, but `See note [reducible non-instances]. -/` has 141 results in 29 files.